drivers/bridge/bridge.go

```diff
@@ -560,6 +603,9 @@ func (d *driver) CreateNetwork(id types.UUID, option map[string]interface{}) err
 
 		// Setup DefaultGatewayIPv6
 		{config.DefaultGatewayIPv6 != nil, setupGatewayIPv6},
+
+		// Add intewr-network communication rules
+		{true, setupNetworkIsolationRules},
```

Comment:
Inter-network communication rules are being installed in the FORWARD chain.
From the discussion we had around having a docker chain per network, I felt a rule that affect inter network should be on a higher level chain.
If you think it should go in the general DOCKER chain, then yes, I need to install these only if set ip tables is true. Not sure what the purpose of that flag originally was. If false then we can't install more than one network.

I ma open to suggestion.